depthai_ros_driver/include/depthai_ros_driver/dai_nodes/imu.hpp

```diff
@@ -0,0 +1,35 @@
+#pragma once
+
+#include "depthai/depthai.hpp"
+#include "depthai_ros_driver/dai_nodes/base_node.hpp"
+#include "depthai_ros_driver/param_handlers/imu_param_handler.hpp"
+#include "rclcpp/rclcpp.hpp"
+#include "sensor_msgs/msg/imu.hpp"
+
+namespace depthai_ros_driver {
+namespace dai_nodes {
+
+class Imu : public BaseNode {
```

Comment:
This should also go under sensors right ?

Reply:
moved to sensors